features/product/managing_price_history/deleting_old_channel_pricing_entry_logs.feature

```diff
@@ -0,0 +1,31 @@
+@managing_price_history
+Feature: Deleting old channel pricing entry logs
+    In order to keep the price history slim
+    As an Administrator
+    I want to delete old channel pricing entry logs
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "PHP T-Shirt"
+        And this product's price has been initially set to "$10.00" on "2022-03-14"
+        And on "2022-03-20" its price changed to "$20.00"
+        And on "2022-03-29" its original price changed to "$25.00"
+        And on "2022-04-14" its price changed to "$15.00" and original price to "$30.00"
+        And on "2022-04-20" its original price has been removed
+        And it is "2022-04-29" now
+
+    @domain
```

Comment:
I wonder about adding also `@cli` tests, but if I see correctly, you've added unit tests, which should be enough

Reply:
Since the command only contains minor user interaction, with all actual logic delegated outside, I didn't see any gain in retesting it with behat. 🤷